# goal definition
geometry_msgs/PoseStamped targetPose
float32 precision
---
# result definition
geometry_msgs/PoseStamped endPose
---
# feedback definition
geometry_msgs/PoseStamped currentPose
